restore/restore.go

```diff
@@ -74,6 +74,11 @@ func RunRestore(ctx context.Context, req Request, svcs Services, log *zap.Logger
 	if err := runner.loadManifests(req.Path); err != nil {
 		return err
 	}
+	// Bail out early if no manifests were found.
+	// TODO: Should this be an error?
```

Comment:
Yeah I think something should be output. Just failing with no output if an invalid path is provided might make me think that everything happened successfully when it had in fact not. Just a simple message like "invalid path" or something like that would be good.